release/lts/nanoserver1909/meta.json

```diff
@@ -11,6 +11,9 @@
         "lts-nanoserver-#shorttag#",
         "lts-nanoserver-1909-#fulltag#"
     ],
+    "manifestLists": [
+        "lts"
```

Comment:
```suggestion
        "nanoserver-lts"
```

Reply:
nano server shouldn't be in the same manifest list.  let's make lts the same a latest and make a new one called nanoserver-lts that is the equivalent of nanoserver

Reply:
I don't quite fully understand the second sentence, but I'll ask about this in the call on Tues. I changed the manifest list values for nanoserver1809, 1909, 2004.